calculator.js

```diff
@@ -0,0 +1,73 @@
+const add = (num1, num2) => num1 + num2
+const subtract = (num1, num2) => num1 - num2
+const multiply = (num1, num2) => num1 * num2
+const divide = (num1, num2) => num1 / num2
+const equalsButton = document.querySelector('button[data-behavior ="equals"]')
+const clearButton = document.querySelector('button[data-behavior ="clear"]')
+let textView = document.querySelector('div[data-behavior ="view"]')
+let calculation = []
+let num1 = ''
+let num2 = ''
+let currentOperator = ''
+
+function equals () {
```

Comment:
I would have this as the last function in the list, as it is generally called last. (maybe before clear?)